src/dashboard/src/templates/archival_storage/list.html

```diff
@@ -25,103 +25,110 @@
 
 {% block content %}
 
-  <ul class="breadcrumb">
-    {% trans "Archival storage" as archival_storage_label %}
-    {% trans "Search" as search_label %}
-    {% breadcrumb_url archival_storage_label 'components.archival_storage.views.overview' %}
-    {% breadcrumb search_label %}
-  </ul>
+  {% if disable_search_indexing %}
+    <h1>Elasticsearch Indexing Disabled</h1>
+    <p class="es-indexing-disabled">Sorry, Elasticsearch indexing has been disabled in this Archivematica installation. The archival storage tab is non-functional when indexing is turned off.</p>
```

Comment:
Fix i18n.

Reply:
Fixed here and also in all newly added template strings.